core/templates/components/summary-tile/story-summary-tile.component.ts

```diff
@@ -59,7 +63,7 @@ export class StorySummaryTileComponent implements OnInit {
     private assetsBackendApiService: AssetsBackendApiService
   ) {}
 
-  getStoryLink(): string {
+  getStoryLink(): string | null {
```

Comment:
Keeping a null as link is dangerous... lets default to '#' in that case.

Reply:
Done, I have also followed this pattern for `assets-backend-api.service` PTAL